client/consensus/grandpa/src/communication/gossip.rs

```diff
@@ -791,68 +791,72 @@ impl<Block: BlockT> Inner<Block> {
 	/// Note a round in the current set has started. Does nothing if the last
 	/// call to the function was with the same `round`.
 	fn note_round(&mut self, round: Round) -> MaybeMessage<Block> {
-		{
-			let local_view = match self.local_view {
-				None => return None,
-				Some(ref mut v) =>
-					if v.round == round {
-						// Do not send neighbor packets out if `round` has not changed ---
-						// such behavior is punishable.
-						return None
-					} else {
-						v
-					},
-			};
+		let local_view = match self.local_view {
+			None => return None,
+			Some(ref mut v) =>
+				if v.round == round {
+					// Do not send neighbor packets out if `round` has not changed ---
+					// such behavior is punishable.
+					return None
+				} else {
+					v
+				},
+		};
 
-			let set_id = local_view.set_id;
+		let set_id = local_view.set_id;
 
-			debug!(
-				target: LOG_TARGET,
-				"Voter {} noting beginning of round {:?} to network.",
-				self.config.name(),
-				(round, set_id)
-			);
+		debug!(
+			target: LOG_TARGET,
+			"Voter {} noting beginning of round {:?} to network.",
+			self.config.name(),
+			(round, set_id)
+		);
 
-			local_view.update_round(round);
+		local_view.update_round(round);
 
-			self.live_topics.push(round, set_id);
-			self.peers.reshuffle();
-		}
-		self.multicast_neighbor_packet()
+		self.live_topics.push(round, set_id);
+		self.peers.reshuffle();
```

Comment:
Nothing changed here, I just removed this useless scope (rustc was dumber when this code was written).